### PR TITLE
[kubernetes] Update to the latest version

### PR DESCRIPTION
**Pull request by ahmedyarub**
**Describe the pull request**

- #### What does your PR fix?  
  Update Kubernetes client library to the latest version, mainly to fix static/dynamic libwebsockets linking

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  all, Yes

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  Yes

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

File: ports/kubernetes/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "name": "kubernetes",
   "version": "0.0.1",
+  "port-version": 1,
```
Comment (JonLiu1993) [inline ports/kubernetes/vcpkg.json:4]:
```suggestion
```

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

File: ports/kubernetes/vcpkg.json (hunk 1 above)
Comment (ahmedyarub) [reply ports/kubernetes/vcpkg.json:4]: Why? This is the second version of the port

**Comment by JonLiu1993** [comment]
@ahmedyarub ,Thanks for your pr, please run the command "./vcpkg x-add-version --overwrite-version" after modify the pr

File: ports/kubernetes/vcpkg.json (hunk 1 above)
Comment (JonLiu1993) [reply ports/kubernetes/vcpkg.json:4]: @ahmedyarub ,Thanks for your pr, we should delete the "port-verison" when update the port version, reference #https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md#update-the-port-version-field-in-the-manifest-file-of-any-modified-ports

Comment (ahmedyarub) [reply ports/kubernetes/vcpkg.json:4]: @JonLiu1993 but I'm not updating the port version, juts the commit. Are you sure that I should delete (or set to 0) the port-version? It is still version 0.0.1 of the library.

Comment (JonLiu1993) [reply ports/kubernetes/vcpkg.json:4]: Yes, if we update the commit,we should delete (or set to 0) the port-version

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

File: ports/kubernetes/vcpkg.json (hunk 1 above)
Comment (ahmedyarub) [reply ports/kubernetes/vcpkg.json]: Done. Thank you for the explanation.

**Comment by JonLiu1993** [comment]
Please run the command "./vcpkg x-add-version kubernetes --overwrite-version" and commit again

**Comment by ahmedyarub** [comment]
> Please run the command "./vcpkg x-add-version kubernetes --overwrite-version" and commit again

Did that but there is nothing to commit.

**Review by JonLiu1993 — APPROVED** [review]

**Review by JonLiu1993 — APPROVED** [review]

**Pull request merged**